src/docs/predicted_molecular_consequence.md

```diff
@@ -8,9 +8,81 @@ A set of information enabling interpretation of likely variant impact on genomic
 | feature_stable_id | string          | Feature (eg transcript) id
 | feature_type      | OntologyTermMetadata        | SO term for feature type (eg. transcript, regulatory feature)
```

Comment:
We proposed that we will always have feature as transcript, just to confirm if we would not have regulatory or motif features in the future? Do we still need `feature_type` if the feature is always transcript?

Reply:
Yes there will be regulatory types. And general intergenic

Reply:
Thanks @jamie-m-a. 

In that case, do we prefer having/retaining the following fields instead of transcript-specific fields?
- feature_stable_id (instead of proposed update to `stable_id`)
- feature_biotype (instead of transcript_biotype)

Reply:
Yeah I think it's better to keep those generic for flexibility.

Reply:
The model for regulatory feature may require capturing of different kind of information like tissue etc, so it has been proposed to have different predicted_molecular_consequence model for different features, hence making it transcript-specific